lib/constants.ts

```diff
@@ -28,11 +28,28 @@ export const developmentPlanTimetableEvents = [
     name: "Plan found unsound",
     key: "plan-found-unsound",
   },
+  {
```

Comment:
I have added this event for when the user selects "Not Adopted by the Local Authority" on "Tell us about the status change" page